src/ocean/util/app/ext/UnixSocketExt.d

```diff
@@ -109,10 +109,13 @@ public class UnixSocketExt : IApplicationExtension, IConfigExtExtension
     public void handle ( cstring command, cstring args,
                          void delegate (cstring) send_response )
     {
+        scope pred = (cstring v) { return v.length; };
```

Comment:
There's no way to inline this into the `filter` call, right? (due to `scope` I assume), since it would make the existence of this clearer.